contributing/documentation.md

```diff
@@ -9,7 +9,7 @@ We autogenerate the documentation of our provider resources and data sources via
 ### How to generate the documentation for a resource
 - Make sure that the resource and data source schemas have defined the fields `MarkdownDescription` and `Description`.
   - We recommend to use [Scaffolding Schema and Model Definitions](#scaffolding-schema-and-model-definitions) to autogenerate the schema via the Open API specification.
-- Add the resource/data source templates to the [templates](templates) folder. See [README.md](templates/README.md) for more info.
+- Add the resource/data source templates to the [templates](templates) folder. See [README.md](./../templates/README.md) for more info.
```

Comment:
the link to the templates folder 404s. I think we need to tweak the relative path
```suggestion
- Add the resource/data source templates to the [templates](./../templates) folder. See [README.md](./../templates/README.md) for more info.
```

Reply:
thanks for calling out, relative paths do not seem to work so went for the full path to master branch.